internal/cli/cmds.go

```diff
@@ -359,7 +359,7 @@ func runRemove(language string, args []string, upgrade bool,
 	}
 
 	s := silenceSubroutines()
-	specfilePkgs := b.ListSpecfile()
+	specfilePkgs := b.ListSpecfile(false)
```

Comment:
why do we need to not merge all groups here?

Reply:
My presumption was that `remove` wouldn't remove from groups that weren't explicitly exposed, but that seems to not be the case. I'll flip this over!

Reply:
```suggestion
	specfilePkgs := b.ListSpecfile(true)
```

Reply:
I tested `npm install --save-dev $pkg && npm remove $pkg` as well as `poetry add -G foo $pkg && poetry remove $pkg`, both passed. If there are edge cases we can pick them up as they come.